### ISSUE TITLE
Official releases of the FASTA database using current time stamp

### ISSUE THREAD

**Issue by peterjc** [grunwaldlab/OomyceteDB#15]
1. Goto http://oomycetedb.cgrb.oregonstate.edu/search.html
2. Click on release 1, dated 2021-03-01, with 885 sequences, comment "First release."
3. Click on "Download database"
4. Observe automatically named file name like ``oomycetedb_whole_2022_07_06_03_05_06.fa``

The system appears to us the current timestamp (probably in your time zone not mine?) giving a filename ``oomycetedb_whole_YYYY_MM_DD_HH_MM_SS.fa`` which is reasonable for a custom search.

However, for a specific release, it would be far preferable to have the release data and number used, e.g. ``omycetedb_whole_2021-03-01_release_1.fa`` in this case. That would be in line with interoperability and reusability aspects of https://en.wikipedia.org/wiki/FAIR_data - reusing the database when the filename is always different is much harder.

**Comment by peterjc** [comment]
I can see that yes, the filename is set to the time stamp in https://github.com/grunwaldlab/OomyceteDB/blob/master/shiny_apps/search/server.r (regardless of what is being downloaded).

**Comment by zachary-foster** [comment]
I agree this would be a good change and we will likely change to use version numbers and/or release dates in future versions

**Comment by peterjc** [comment]
Perfect - I'll look forward to that. Thank you.

**Comment by peterjc** [comment]
The website page https://grunwaldlab.github.io/OomyceteDB/search.html now points to the v1 and v2 releases as FASTA files, hosted on GitHub from https://github.com/grunwaldlab/OomyceteDB/tree/master/data/releases - that resolves my request, thank you. Closing issue.
